docs/_data/portland-2022-config.yaml

```diff
@@ -210,15 +210,15 @@ sponsors:
 
 
 # Things that change over time, listed in order of change
```

Comment:
The hike flag is on under "truthy things that don't change" and there are mentions of it on the website, but I don't know whether to delete the actual text or turn off the flag. Halp!

Reply:
I think False works fine.